packages/module-manager/package.json

```diff
@@ -39,5 +39,8 @@
     "typescript": "5.2.2",
     "vite": "^5.0.0",
     "vite-plugin-dts": "3.6.4"
+  },
+  "dependencies": {
+    "dist": "link:/home/hxsggsz/frontend/octopost/packages/module-manager/dist"
```

Comment:
pq essa dist puxa especificamente o path do seu pc?

Reply:
isso é coisa do pnpm, eu to linkando esse package `module-manager` num modulo fake pra testar sem precisar subir esse pacote no npm pra usar ele